src/lib/viewers/controls/media/MediaSettings.tsx

```diff
@@ -74,6 +91,13 @@ export default function MediaSettings({
             {quality && onQualityChange && (
                 <MediaSettingsMenuQuality onQualityChange={onQualityChange} quality={quality} />
             )}
+            {showSubtitles && onSubtitleChange && (
```

Comment:
Can these various submenus decide to not render themselves if these props are missing?